pkgs/by-name/qs/qsv/package.nix

```
@@ -21,19 +21,11 @@ rustPlatform.buildRustPackage {
     owner = "jqnatividad";
     repo = "qsv";
     rev = version;
-    hash = "sha256-erXMDZBkOPnZdhhjnUTiS0eCuFo1v5sNhLn8o7QN/5g=";
+    hash = "sha256-27oSk8fnTvl1zJ8xYkZHkWVTq+AVDn4Zi1s56T49T1Q=";
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "calamine-0.25.0" = "sha256-LFQahQ+SfGWhp5Kfs7AJCj4zyxWiz099pwVuQucCF00=";
-      "dynfmt-0.1.5" = "sha256-/SrNOOQJW3XFZOPL/mzdOBppVCaJNNyGBuJumQGx6sA=";
-      "grex-1.4.5" = "sha256-4Tr5L87HuiUW8tJdqr4oT1yQXviU7XtDrKY7iYNcwbo=";
-      "local-encoding-0.2.0" = "sha256-ThXYKr3u/n2kvINcyobB2Ayex2sNbJEOyyjZH993Z4U=";
-      "polars-0.41.3" = "sha256-8xkcJgfKo7BzeBnR6XVdbcH9ZY9Kh4dcGFMmyA5LuQg=";
-    };
-  };
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-tu9HCFAxmmYVgmJyHunBtGSqKGzwbX2vi6ju4cv33wc=";
 
   buildInputs =
```

```suggestion
  buildInputs = [
    file
    sqlite
    zstd
  ];
```

Sorry for another non-version related change, just good to simplify this package I think. See https://github.com/NixOS/nixpkgs/issues/354146, we no longer need to specify darwin framework to include!